io.github.nemuelw.protodesk.yml

```diff
@@ -0,0 +1,66 @@
+app-id: io.github.nemuelw.protodesk
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: protodesk
+
+finish-args:
+  - --filesystem=host
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: kerberos
+    subdir: src
+    config-opts:
+      - --disable-static
+      - --disable-rpath
+    sources:
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.21/krb5-1.21.2.tar.gz
+        sha256: 9560941a9d843c0243a71b17a7ac6fe31c7cebb5bce3983db79e52ae7e850491
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /sbin
+      - /share/et
+      - /share/examples
+
+  - name: Protodesk
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/nemuelw/protodesk/73e45c4ec2b8f64c82fdd35dae90c49ba77e5024/flathub/io.github.nemuelw.protodesk.desktop
+        sha512: d9804756efcfc05e7827097f235fdd59b918ad3d390abb3f099f24c02a16c158eab3757b0b5d937ee29a0d1757f2897d78352f6fd8f26e60aaf7a4b7418efa99
+        dest: ./share/applications/io.github.nemuelw.protodesk.desktop
+
+      - type: file
+        url: https://raw.githubusercontent.com/nemuelw/protodesk/e98976440e3ea147f77f9fc46e468b9d82791701/flathub/io.github.nemuelw.protodesk.metainfo.xml
+        sha512: 7c74e36730a732da9a9fb09e3ecc7d15b8a77ae2f8acf8529d967c3dd1c74dca4fa96863a0b9a67f8991e2d8432240d43a3d7f2f761c2a66da134dc46ee709c9
+        dest: ./share/metainfo/io.github.nemuelw.protodesk.metainfo.xml
+
+      - type: file
+        url: https://raw.githubusercontent.com/nemuelw/protodesk/73e45c4ec2b8f64c82fdd35dae90c49ba77e5024/assets/logo.svg
+        sha512: 1ea4475c7673457fd1c8c8d24089fddebe9653081c3db30d93a86b2e71284470bd04ad987632214e0030c4a381700c556747aa93c867545e9cf494b259b71003
+        dest: ./share/icons/hicolor/scalable/apps/io.github.nemuelw.protodesk.svg
+
+      - type: archive
+        url: https://github.com/nemuelw/protodesk/releases/download/v1.2.6/protodesk-v1.2.6_amd64.tar.gz
```

Comment:
why not building from source instead?

Reply:
The app will be distributed via other channels besides Flathub, and so having a centralized upstream build is the better option.

Reply:
That's a myth. Also if it's so centralized with doesn't it include icon, desktop file and appstream file?

Also I see it does include Qt which is a bad idea. The runtime has patches in Qt to work better with Flatpak and Linux in general. Instead you bloat the distribution.

Reply:
As you might have seen, the flatpak specific files are in their own directory. Upon having the builds upstream, it's more straightforward to also create deb, rpm or snap.

Reply:
neither of these files are flatpak specific. They work for any linux distro

and they are not even installed.

Reply:
I was referring to the files in the [`flathub` directory](https://github.com/nemuelw/protodesk/tree/main/flathub) specifically.

Reply:
as I did.

Reply:
Gotcha! Thanks for the clarification.

Reply:
Anything else I might be missing here?

Reply:
> and they are not even installed.

as I said, but I guess it needed to be demonstrated:

```
        "appstream-metainfo-missing: No metainfo file for io.github.nemuelw.protodesk was found in /app/share/metainfo or /app/share/appdata"
```